Vagrantfile

```diff
@@ -61,6 +61,7 @@ ansible_provision = proc do |ansible|
       ceph_mon_docker_subnet: "#{SUBNET}.0/24",
       ceph_osd_docker_extra_env: "CEPH_DAEMON=OSD_CEPH_DISK,OSD_JOURNAL_SIZE=100",
       ceph_osd_docker_devices: settings['disks'],
+      ceph_docker_on_openstack: OSVM,
```

Comment:
We should not assume that we are running this on openstack by default.
So either you create a new condition for this or you update your `group_vars/all` file.